libs/ptr/ptr.go

```diff
@@ -33,3 +33,7 @@ func StringOr(s *string, or string) string {
 func FromTime(t time.Time) *time.Time {
 	return &t
 }
+
+func To[T any](v T) *T {
+	return &v
+}
```

Comment:
While I can see how it was a logical continuation of this package, it's very existence goes against the idea of loose coupling by having small isolated chunks of code independently.

Provided that this is a separate module, the coupling becomes even more tight, as there is now an even tighter dependency.

I would be in faviour of keeping things such as `ptrTo` where they are used, even though they might be needed in multiple places.